### PR TITLE
New blog post proxysql query mirroring

### PR DESCRIPTION
**Pull request by IzzySmith**
This PR adds a new author, and a new blog post about ProxySQL query mirroring.

### CODE REVIEW

File: content/blog/2024/2024-04-23-proxysql-query-mirroring.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+---
+title: "Using ProxySQL Query Mirroring to test query performance on a new cluster"
+date: "2024-04-23T00:00:00+00:00"
+draft: false
+tags: ["ProxySQL", "Upgrades"]
```
Comment (IzzySmith) [inline content/blog/2024/2024-04-23-proxysql-query-mirroring.md:5]: What is the guidance regarding image thumbnails for blogposts?

File: content/blog/2024/2024-04-23-proxysql-query-mirroring.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+It is worth noting, that you should allow enough time for the MySQL buffer pool to get filled, before checking the `stats_mysql_query_digest` table. Otherwise, the query times on the new cluster can be skewed, as the active dataset may not yet be in memory (whereas on the current cluster it might be). Also, keep in mind that if you are mirroring only a subset of queries, the load on the new cluster will be different to the current cluster, and could affect the query performance on the new cluster, so that they appear significantly faster. Checking the execution plan of the query to see whether it has changed, is therefore more important than looking at overall load.
+
+To conclude, using query mirroring to test queries on a new system, before making the migration, allows you to compare latency and query plans per normalised query, and proactively detect any necessary alterations before switching live traffic to the new cluster. Now that the servers have been configured in ProxySQL, you can perform the failover within ProxySQL, without needing to adjust the application. More details on that, are planned for an upcoming blog post.
```
Comment (IzzySmith) [inline content/blog/2024/2024-04-23-proxysql-query-mirroring.md:142]: > More details on that, are planned for an upcoming blog post.

Should I mention this already, or wait until I have written that blog post, and link it, once it has been approved?

File: content/blog/2024/2024-04-23-proxysql-query-mirroring.md (hunk 1 above)
Comment (dbazhenov) [reply content/blog/2024/2024-04-23-proxysql-query-mirroring.md:5]: @IzzySmith 
place the images in the directory assets/blog/2024/04/
use Alt

File: content/blog/2024/2024-04-23-proxysql-query-mirroring.md (hunk 2 above)
Comment (dbazhenov) [reply content/blog/2024/2024-04-23-proxysql-query-mirroring.md:142]: As you like, you can edit later.

File: content/blog/2024/2024-04-23-proxysql-query-mirroring.md (hunk 1 above)
Comment (isobelsmith) [reply content/blog/2024/2024-04-23-proxysql-query-mirroring.md:5]: Hi @dbazhenov, 

Thanks for your reply. Sorry I meant to ask: what tool do you use to generate the images on the blog / where do you source them from? I am struggling to find something that looks as good as the image thumbnails used on other articles.

File: content/blog/2024/2024-04-23-proxysql-query-mirroring.md (hunk 2 above)
Comment (isobelsmith) [reply content/blog/2024/2024-04-23-proxysql-query-mirroring.md:142]: Thanks :) I will remove it, and add the link, once the other post has been written.

File: content/blog/2024/2024-04-23-proxysql-query-mirroring.md (hunk 1 above)
Comment (IzzySmith) [reply content/blog/2024/2024-04-23-proxysql-query-mirroring.md:5]: I managed to generate something fancy with DALL-E, so I will mark this as resolved

**Review by dbazhenov — APPROVED** [review]
Great! Thank you!

**Comment by dbazhenov** [comment]
@IzzySmith 
The post has been published! Congratulations!!! Thank you for your contribution
https://percona.community/blog/2024/05/01/using-proxysql-query-mirroring-to-test-query-peromance-on-a-new-cluster/

File: content/blog/2024/2024-04-23-proxysql-query-mirroring.md (hunk 1 above)
Comment (edithturn) [reply content/blog/2024/2024-04-23-proxysql-query-mirroring.md:5]: Hi, @IzzySmith @dbazhenov, for images like backgrounds or landscapes, I used ChatGPT but did not get good pictures from them. 
For my diagrams I use: https://excalidraw.com/
For my design I use Canva: https://www.canva.com/

Comment (isobelsmith) [reply content/blog/2024/2024-04-23-proxysql-query-mirroring.md:5]: Thankyou so much for your suggestions :)

**Comment by isobelsmith** [comment]
> @IzzySmith The post has been published! Congratulations!!! Thank you for your contribution https://percona.community/blog/2024/05/01/using-proxysql-query-mirroring-to-test-query-peromance-on-a-new-cluster/

Thanks so much for your help and support!

**Pull request merged**